<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pattern_menu</class>
 <widget class="QWidget" name="pattern_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 44, 54);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="styleSheet">
      <string notr="true">QPushButton:enabled {
	color: rgb(225, 230, 241);
	font: 12px  &quot;游ゴシック&quot;;
	border:none;
	Text-align:left;
}
QPushButton:disabled {
	color: rgb(99, 99, 99);
	border:none;
	Text-align:left;
}
QPushButton:hover {
	background-color: rgb(29, 33, 40);
}

QPushButton:pressed {
	background-color: rgb(69, 140, 207);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="8" column="0">
       <widget class="QPushButton" name="pattern_menu_leftinsertblank_pushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>左に新規挿入</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pattern_menu_leftinsert_pushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>左に挿入</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pattern_menu_copy_pushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>コピー</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pattern_menu_rightinsertblank_pushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>右に新規挿入</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(141, 145, 156);
}
QFrame:hover{
	border:none;
}
QFrame:focus{
	border:none;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QFrame" name="frame_7">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(141, 145, 156);
}
QFrame:hover{
	border:none;
}
QFrame:focus{
	border:none;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pattern_menu_rightinsert_pushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>右に挿入</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QFrame" name="frame_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>2</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(141, 145, 156);
}
QFrame:hover{
	border:none;
}
QFrame:focus{
	border:none;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pattern_menu_paste_pushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>貼り付け</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="pattern_menu_delete_pushButton">
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {
	color: rgb(255, 88, 105);
}
QPushButton:disabled {
	color: rgb(99, 99, 99);
}</string>
        </property>
        <property name="text">
         <string>削除</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pattern_menu_cut_pushButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>切り取り</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
